### PR TITLE
Feat/buttonhover

### PR DESCRIPTION
**Pull request by AdityaAB4**
# Issue #2📐

- **I [Aditya](@AdityaAB4) have worked for Issue #2**

[put x to check the boxes]: <> (This is a comment, it will not be included)

## Guidelines 🔐

**I accept the fact that I have followed the guidelines and have not copied the codes from around the internet**

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- **My pull request closes Issue #2**

## Screenshots 📷

**Here are the pictures of the changes that I have made 🔽**

After hover : 
image

  
---

### CODE REVIEW

File: src/components/Button.jsx [hunk 1]
```diff
@@ -1,16 +1,50 @@
-const Button = ({ children, icon, backgroundColor, textColor, borderColor, fullWidth }) => {
-	return (
-		<button
-			className={`flex justify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full ${
-				backgroundColor ? `${backgroundColor} ${borderColor} ${textColor}` : "bg-coral-red  border-coral-red text-white"
-			}
-			${fullWidth && 'w-full'}
+const Button = ({
+  children,
```
Comment (warmachine028) [inline src/components/Button.jsx:2]: - We do not recommend using direct CSS styling for this project.
- Read the Tailwind CSS docs and then add hover effects to this button component

**Review by warmachine028 — CHANGES_REQUESTED** [review]

File: src/components/Button.jsx (hunk 1 above)
Comment (AdityaAB4) [reply src/components/Button.jsx:2]: I have added the hover effects according to Tailwind CSS.

File: src/components/Button.jsx [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+  backgroundColor,
+  borderColor,
+  textColor,
+  fullWidth,
+  icon,
+}) => {
+  return (
+    <>
+      <button
+        className={`flex buttonjustify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full hover:bg-white hover:text-coral-red hover:border-coral-red ${
```
Comment (warmachine028) [inline src/components/Button.jsx:12]: I think there is a small error in this line, `flex buttonjustify ...`, this is a typo I presume. Please fix this

**Review by warmachine028 — CHANGES_REQUESTED** [review]

File: src/components/Button.jsx (hunk 1 above)
Comment (warmachine028) [reply src/components/Button.jsx:2]: Great work!

File: src/components/Button.jsx [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
+const Button = ({
+  children,
+  backgroundColor,
+  borderColor,
+  textColor,
+  fullWidth,
+  icon,
+}) => {
+  return (
+    <>
```
Comment (warmachine028) [inline src/components/Button.jsx:10]: Also remove this ```<></>``` element, believing this is also not required

**Comment by AdityaAB4** [comment]
I have made the changes and removed the typo

**Review by warmachine028 — APPROVED** [review]
Great work! All problems seem to be fixed.

**Comment by warmachine028** [comment]
## Thanks for your valuable contribution to this Project 🎊🎊

**Comment by warmachine028** [comment]
- Make sure to star this Project ⭐⭐⭐
- Check out other projects [here](https://github.com/warmachine028?tab=repositories)

**Pull request merged**